joomlagerman/joomla
# [4.0] Add script.php for the german language package

Pull Request für Issue https://github.com/joomlagerman/joomla/issues/1915

### Zusammenfassung der Änderungen

- [x] Create a script.php to remove the renamed and removed language files. For exampe the de-DE.com_config.ini (as well as de-XX).
- [x] And add a check that the 4.x pack can not be installed in 3.x

### How to test

Install 4.0
install 3.xer language pack: https://github.com/joomlagerman/joomla/releases/download/3.9.28v1/de-DE_joomla_lang_full_3.9.28v1.zip
Check that the "old" files like de-DE.com_config.ini is there.
Install 4.xer language pack build with the changes here: 
[de-DE_joomla_lang_full_4.0.0-rc4v2.zip](https://github.com/joomlagerman/joomla/files/6946187/de-DE_joomla_lang_full_4.0.0-rc4v2.zip)
check that the de-DE.com_config is gone.


Try to install the 4.0er language pack into 3.9 or 3.10-dev -> error message should be shown.
